Modules/System/MicrosoftGraph/src/helper/MicrosoftGraphUriBuilder.Codeunit.al

```diff
@@ -0,0 +1,56 @@
+// ------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// ------------------------------------------------------------------------------------------------
+
+codeunit 9152 "Microsoft Graph Uri Builder"
+{
+    Access = Internal;
+    InherentEntitlements = X;
+    InherentPermissions = X;
+
+    var
+        Uri: Codeunit Uri;
+        MicrosoftGraphAPIVersion: Enum "Microsoft Graph API Version";
+        BaseUriLbl: Label 'https://graph.microsoft.com/%1/', Comment = '%1 = Graph API Version', Locked = true;
```

Comment:
Do we need some kind of providing a differnt Base URI.
e.g. `https://graph.microsoft-ppe.com`?